### DISCUSSION TITLE
日语汉字的音标不对

### DISCUSSION THREAD

**Discussion by freeyob** [ZuodaoTech/everyone-can-use-english#992] [Q&A] [answered]
请问支持日语不，我发现日语中汉字的音标不对，使用的是wisper。所有汉字都被替换成了"tʃˈaɪniːzlˈe̞tə"chinese letter。
<img width="820" alt="Screenshot 2024-08-18 at 9 30 35 PM" src="https://github.com/user-attachments/assets/988b69b2-8fe2-47dd-9541-7e8edb170a58">

**Comment by an-lee** [comment, marked as answer]
目前日语的音标确实不对。可以先忽略吧
